docs/pages/buttons.vue

```diff
@@ -65,11 +67,15 @@
       </p>
 
       <DocsShow>
-        <KButton text="Primary" :primary="true" appearance="raised-button" />
-        <KButton text="Secondary" :primary="false" appearance="raised-button" />
+        <KButtonGroup style="margin-bottom: 8px;">
+          <KButton text="Primary" :primary="true" appearance="raised-button" />
+          <KButton text="Secondary" :primary="false" appearance="raised-button" />
+        </KButtonGroup>
         <br>
-        <KButton text="Primary" :primary="true" appearance="flat-button" />
-        <KButton text="Secondary" :primary="false" appearance="flat-button" />
+        <KButtonGroup>
+          <KButton text="Primary" :primary="true" appearance="flat-button" />
+          <KButton text="Secondary" :primary="false" appearance="flat-button" />
+        </KButtonGroup>
```

Comment:
could we add the icon button, e.g.:

![image](https://user-images.githubusercontent.com/2367265/88116398-4c49cc00-cb6d-11ea-9e6a-0052468a2838.png)